app/controllers/admin/snap_applications_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Admin
+  class SnapApplicationsController < Admin::ApplicationController
+    # To customize the behavior of this controller,
```

Comment:
For the controller, I think it's worthwhile to keep these in. It's the one layer in this gem that I end up touching more often than the others so it's helpful to know exactly what/where changes are often made.